cnd/src/swap_protocols/rfc003/actions/ether.rs

```diff
@@ -14,11 +14,13 @@ use crate::{
 };
 use blockchain_contracts::ethereum::rfc003::ether_htlc::EtherHtlc;
 
-impl FundAction<Ethereum, asset::Ether> for (Ethereum, asset::Ether) {
+impl FundAction<Ethereum, asset::Ether, identity::Ethereum>
+    for (Ethereum, asset::Ether, identity::Ethereum)
```

Comment:
Same here, we shouldn't need this:
```suggestion
    for (Ethereum, asset::Ether)
```